courses/views.py

```diff
@@ -122,6 +122,12 @@ def user_existence(request):
 def create_ccx(request):
     """
     Supports creating a CCX on edx.
+
+    Args:
```

Comment:
Can you change `Args` to `Request parameters`, so that this doesn't look like a napoleon docstring?